include/fb_globals.h

```diff
@@ -132,6 +136,9 @@ extern gedict_t *dropper;
 #define BOTPATH_CURLJUMP_HINT		(1 << 23)
 #define BOTPATH_FULL_AIRCONTROL		(1 << 24)
 #define BOTPATH_RJ_IN_PROGRESS		(1 << 25)
+#define HOOK                        (1 << 26)
+#define LOOK_BUTTON                 (1 << 27)	// Indicates path which points to a button to shoot 
+#define FIRE_BUTTON                 (1 << 28)   // Set when a bot should try shooting a button
```

Comment:
Perhaps move LOOK_BUTTON and FIRE_BUTTON to 26 resp 27 (or I guess leave the 26 hook gap as it will be filled by this PR later), and open a separate PR for adding these to trim down the PR size given that they provide standalone functionality to bots.